docs/source/glossaries.rst

```diff
@@ -0,0 +1,107 @@
+**********
+Glossaries
+**********
+
+=========== ====================================================================================================================================================
+Keyword     Description
+=========== ====================================================================================================================================================
+ACK         Acknowledge
+am          Active Message
```

Comment:
AM (pls use upper case for all acronyms)

Reply:
done. I modified some words to uppercase. Please take a look again.